microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -44,7 +44,9 @@ To use the Microsoft Edge Add-ons API, you need to enable the API for your proje
 
 1. In the **Publish APIs** page, select the **Create API credentials** button to generate the API credentials.  This step may take a few minutes.  After the APIs are enabled, the **Client ID**, **Client Secret**, and **Auth Token URL** are displayed on this page.
 
-1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**.  You'll use them in the next step to get an access token.
+1. Note the **ClientID**, **Client Secret** and the **Auth Token URL**. You'll use them in the next step to get an access token. Note that client secret is visible only once, after enabling or renewing the API. 
+
```

Comment:
"The client secret is visible only once, after enabling or renewing the API."
Not sure that statement makes sense - or maybe it's clear, in context in the rendered page:
https://review.docs.microsoft.com/en-us/microsoft-edge/extensions-chromium/publish/api/using-addons-api?branch=pr-en-us-1694#before-you-begin
The 2nd part seems like it might be confusing.  Do we need to clarify or strengthen the statement?  After you enable the API or renew the API, then the client secret is shown to you one time only, at that time only?  What can you not do?  Might need to add a concrete, specific "you can't do this" scenario description:
After you enable or renew the API, and do xyz, you can no longer view the client secret.